protos/in_toto_attestation/predicates/vsa/v0_2/vsa.proto

```diff
@@ -1,10 +1,10 @@
 syntax = "proto3";
 
-package io.in_toto.attestation.predicates;
+package in_toto_attestation.predicates;
 
 import "google/protobuf/timestamp.proto";
 
-option go_package = "github.com/in-toto/attestation/go/spec/predicates";
+option go_package = "github.com/in-toto/attestation/go/predicates";
```

Comment:
```suggestion
option go_package = "github.com/in-toto/attestation/go/predicates/vsa/v0";
```

I don't know if we've taken a call on the versioning but we should mark separate packages for each predicate IMO.

Reply:
If we're versioning by dir, `message VerificationSummaryV02 {` can be changed to drop the version.

Reply:
ping @TomHennen for this as this might be what the slsa-verifier expects.

Reply:
I think @laurentsimon knows best (and is working on VSA features now).  WDYT Laurent?

Reply:
I've been waiting for the v1.0 cleanups on VSA to add support in slsa-verifier, so no particular requirement / opinion from me.

Reply:
In that case I don't believe anyone is depending on this behavior yet.